dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ResourcesController.java

```diff
@@ -146,7 +151,7 @@ public Result<Object> createDirectory(@ApiIgnore @RequestAttribute(value = Const
     @PostMapping()
     @ApiException(CREATE_RESOURCE_ERROR)
     @AccessLogAnnotation(ignoreRequestArgs = "loginUser")
-    public Result<Object> createResource(@ApiIgnore @RequestAttribute(value = Constants.SESSION_USER) User loginUser,
+    public Result<org.apache.dolphinscheduler.dao.entity.Resource> createResource(@ApiIgnore @RequestAttribute(value = Constants.SESSION_USER) User loginUser,
```

Comment:
```suggestion
    public Result<Resource> createResource(@ApiIgnore @RequestAttribute(value = Constants.SESSION_USER) User loginUser,
```

Reply:
this Resource is duplicated with `org.springframework.core.io.Resource` in method downloadResource。 and this Controller is  assigned with Deprecated。so haven’t do anything about this。